app/views/candidates/home/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">Request school experience placements</h1>
+
+    <p class="govuk-body-lead">
+      Use this service if you're interested in becoming a teacher and want
+      to find and request school experience placements in schools across
+      England.
+    </p>
+
+    <p>
```

Comment:
Paragraphs are required to have the class `govuk-body`. However, I can't see any actual change in appearance if it's missing.